app/subscribers/solidus_drip/order_updater_subscriber.rb

```diff
@@ -0,0 +1,18 @@
+module SolidusDrip
+  module OrderUpdaterSubscriber
+    include ::Spree::Event::Subscriber
+
+    event_action :order_recalculated
```

Comment:
One problem with `order_recalculated` is that it's fired _before_ the totals are persisted to the DB. I'm not sure of the details of this extension, so this might be okay, but you might want to keep in mind that this logic will fire when the order hasn't been saved yet.

Reply:
Thanks for pointing this out, I didn't think about this specific thing. Checking the source code, it seems that the previous [update-hook version](https://github.com/solidusio/solidus/blob/v2.10/core/app/models/spree/order_updater.rb#L29) was triggered before save as well though. I think we are good.